README.md

```diff
@@ -1,5 +1,5 @@
 # Presto
-[![Build Status](https://github.com/prestosql/presto/workflows/CI/badge.svg)](https://github.com/prestosql/presto/actions?query=workflow%3ACI+event%3Apush+branch%3Amaster)
+[![Build Status](https://github.com/prestosql/presto/workflows/checks/badge.svg?branch=master&event=push)](https://github.com/prestosql/presto/actions?query=workflow%3Achecks+event%3Apush+branch%3Amaster) [![Presto Slack](https://img.shields.io/static/v1?logo=slack&logoColor=959DA5&label=Slack&labelColor=333a41&message=join%20conversation&color=3AC358)](https://prestosql.io/slack.html)
```

Comment:
Add a newline in the source to separate each badge. They'll still be rendered on one line. (see Airlift readme)

Reply:
i got misled by intellij's renderer, thanks